Modules/Tracker/TrackerBaseFrame.lua

```diff
@@ -349,39 +356,66 @@ function TrackerBaseFrame.ShrinkToMinSize(minSize)
+        GameTooltip:Hide()
+        GameTooltip._SizerToolTip = nil
     end
 
-    if IsMouseButtonDown(button) then
+    if InCombatLockdown() or IsShiftKeyDown() or IsAltKeyDown() then
+        Questie:Debug(Questie.DEBUG_DEVELOP, "[TrackerBaseFrame:OnDragStart] - In Combat or shift key or alt key detected! --> Exiting.")
+        return
+    else
         if (IsControlKeyDown() and Questie.db.global.trackerLocked and not ChatEdit_GetActiveWindow()) or not Questie.db.global.trackerLocked then
-            dragButton = button
-            baseFrame:StartMoving()
-
-            if GameTooltip:IsShown() then
-                GameTooltip:Hide()
+            if TrackerBaseFrame.isMoving ~= false or TrackerBaseFrame.isSizing == true then
+                Questie:Debug(Questie.DEBUG_DEVELOP, "[TrackerBaseFrame:OnDragStart] - Frame is already moving or frame is already resizing! --> Exiting.")
+                return
             end
-
-            TrackerBaseFrame:Update()
         else
-            -- Turns off mouse looking to prevent frame from becoming stuck to the pointer
-            if not IsMouselooking() then
-                MouselookStart()
-                mouseLookTicker = C_Timer.NewTicker(0.1, function()
-                    if not IsMouseButtonDown(button) then
-                        MouselookStop()
-                        mouseLookTicker:Cancel()
-                    end
-                end)
+            Questie:Debug(Questie.DEBUG_DEVELOP, "[TrackerBaseFrame:OnDragStart] - Tracker is Locked. Use CTRL Key. --> Exiting.")
+            return
+        end
+    end
+
+    if TrackerBaseFrame.isMoving ~= true and TrackerBaseFrame.isSizing ~= true then
+        if IsMouseButtonDown(button) and button ~= "MiddleButton" then
+            if (IsControlKeyDown() and Questie.db.global.trackerLocked and not ChatEdit_GetActiveWindow()) or not Questie.db.global.trackerLocked then
+                if baseFrame:IsMovable() then
+                    Questie:Debug(Questie.DEBUG_DEVELOP, "[TrackerBaseFrame:OnDragStart] - Dragging Started.")
+                    TrackerBaseFrame.isMoving = true
+
+                    baseFrame:StartMoving()
+                    TrackerBaseFrame:Update()
+                    --[[
+                    -- If we're filtering the Middle Button then theoretically we don't need this.
+                    else
+                        -- Turns off mouse looking to prevent frame from becoming stuck to the pointer
+                        if not IsMouselooking() then
+                            MouselookStart()
+                            mouseLookTicker = C_Timer.NewTicker(0.1, function()
+                                if not IsMouseButtonDown(button) then
+                                    MouselookStop()
+                                    mouseLookTicker:Cancel()
+                                end
+                            end)
+                        end
+                    --]]
```

Comment:
Do we need this or not?

Reply:
I left it in for a few more days of testing. It's not needed anymore. I think it was a REALLY old blizzard bug back in the early days of classic.